src/encoded/schemas/target.json

```diff
@@ -38,17 +38,39 @@
             "type": "string",
             "linkTo": "Organism"
         },
+        "targeted_genes": {
+            "title": "Targeted genes",
+            "description": "List of gene(s) that are targeted by corrsponding assay or antibody.",
+            "comment": "See gene.json for available identifiers and details.",
+            "type": "array",
+            "uniqueItems": true,
+            "default": [],
+            "items": {
+                "title": "Targeted Gene",
+                "description": "A gene that are targeted by corrsponding assay or antibody.",
+                "comment": "See gene.json for available identifiers.",
+                "type": "string",
+                "linkTo": "Gene"
+            }
+        },
         "gene_name": {
             "title": "Gene name",
             "description": "HGNC or MGI identifier for the target.",
             "comment": "Submit only the identifier (e.g. HMFN0395 or 22809).",
             "type": "string",
             "format": "gene_name"
         },
+        "modification": {
+            "title": "Modification to the corresponding gene.",
+            "description": "A textual description of modification(s) made to the corresponding gene in this targets (e.g. epitope tag(s), post-translational modifications)",
+            "comment": "This property will be used to calculate the proper label of the target. See computed_label property for details.",
+            "type": "string",
+            "default": ""
+        },
         "label": {
-            "title": "Common name with modification",
-            "description": "Common name for the target including post-translational modifications, if any.",
-            "comment": "Submit the common name of the gene with modification (e.g. H3K4me3, eGFP-E2F1, or POLR2AphosphoS2).",
+            "title": "Preferred target label",
+            "description": "An easy to understand label preferred over  computed label.",
+            "comment": "Do not submit. DCC personnel is responsible for submission of this property. In general, the format is the common name of the gene with modification (e.g. H3K4me3, eGFP-E2F1, or POLR2AphosphoS2).",
```

Comment:
This comment is not enforced by import_items. Not sure if it should be but the description doesn't match what will be enforced.

Reply:
it should be import-items.

Reply:
For this comment and the comment below, should the whole target object be admin only? I think @hitz mentioned nobody really post target objects.

Reply:
Well, if you made this change then that effectively makes the whole target object admin only. We can try it and see if we get an avalanche of requests for new ones. I think it will be unlikely but we can always relax it later.